jsemanticdb/src/main/protobuf/bazel_query.proto

```
@@ -0,0 +1,44 @@
+// A trimmed subset of Bazel's `src/main/protobuf/build.proto` for the MBT
+// importer. Field NUMBERS must stay identical to build.proto — that is what the
+// wire format depends on.
+// Based on
+// https://github.com/bazelbuild/bazel/blob/master/src/main/protobuf/build.proto
+syntax = "proto2";
+
+package metals_bazel_query;
```

Buf (1.71.0)

[error] Files with package "metals_bazel_query" must be within a directory "metals_bazel_query" relative to root but were in directory "jsemanticdb/src/main/protobuf".

(PACKAGE_DIRECTORY_MATCH)